be/src/exec/pipeline/pipeline_builder.cpp

```diff
@@ -120,6 +120,35 @@ OpFactories PipelineBuilderContext::_maybe_interpolate_local_passthrough_exchang
     return {std::move(local_exchange_source)};
 }
 
+void PipelineBuilderContext::maybe_interpolate_local_passthrough_exchange_for_sink(RuntimeState* state,
+                                                                                   OpFactoryPtr table_sink_operator,
+                                                                                   int32_t source_operator_dop,
+                                                                                   int32_t desired_sink_dop) {
```

Comment:
Is it possible to return directly when `source_operator_dop` is equal to `desired_sink_dop`?